lib/apimatic-core/utilities/date_time_helper.rb

```diff
@@ -122,5 +122,52 @@ def self.from_rfc3339(date_time)
         DateTime.rfc3339("#{date_time}Z")
       end
     end
+
+    def self.validate_datetime(dt_format, dt)
```

Comment:
Nits, Non-blocker: This method named could be renamed to `valid_datetime?`.

Reply:
done